src/IceRpc.Protobuf.Generators/Internal/DiagnosticDescriptors.cs

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) ZeroC, Inc.
+
+using Microsoft.CodeAnalysis;
+
+namespace IceRpc.Protobuf.Generators.Internal;
+
+internal static class DiagnosticDescriptors
+{
+    internal static DiagnosticDescriptor DuplicateOperationNames { get; } = new DiagnosticDescriptor(
```

Comment:
How is this possible?

Reply:
You can implement several Protobuf interfaces in the same dispatcher, which can contain duplicate operation names.

Reply:
Ok